bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -18,4 +18,6 @@ Import-Package: groovy.lang,
  org.junit,
  org.junit.runner,
  org.junit.runners,
- org.osgi.service.cm
+ org.osgi.service.cm,
+ org.mockito
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
You use `import static org.mockito.Mockito.*;` and call e.g. `when(...)`

The function you are calling is defined in `org.mockito.Mockito` as followed:

```java
    public static <T> OngoingStubbing<T> when(T methodCall) {
        return MOCKITO_CORE.when(methodCall);
    }
```

So, there is e.g. `org.mockito.stubbing.OngoingStubbing<T>` used.
The package `org.mockito.stubbing` is not part of the respective classloader, because there is no import section defined.
If you use Require-Bundle all part of the required bundle is available in by the classloader, but you refer a special bundle and not "any" bundle that provides that package.


> This affects some other places too where mockito was introduced recently. I will push another PR addressing this.

There should be no other place affected, IIRC "do not use Require-Bundle" is part of our coding guidelines :wink: 
So, if it has been used by new contributions, it would be great, if you could remove it again.